src/components/styles/common.scss

```diff
@@ -0,0 +1,18 @@
+@keyframes co-spin {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.co-spin {
```

Comment:
Don't redefine the same CSS as console provides. I'd avoid adding any CSS with a `co-` prefix and instead use your plugin name as a prefix.